merkle_zeppelin/data_io/json_exporter.py

```diff
@@ -0,0 +1,8 @@
+from .base_exporter import MerkleTreeExporter
+from .dto import MerkleTreeDTO
+
+
+class MerkleTreeJSONExporter(MerkleTreeExporter):
+    @staticmethod
+    def export_tree(data: MerkleTreeDTO) -> str:
+        return data.model_dump_json(by_alias=True)
```

Comment:
pydantic `model_dump_json` and return type str ? 🤔

Reply:
```
>>> leaf.model_dump(by_alias=True)
{'value': (1,), 'treeIndex': 1}
>>> leaf.model_dump_json(by_alias=True)
'{"value":[1],"treeIndex":1}'
```
yes, indeed, counterintuitive but it works like above